### DISCUSSION TITLE
Ubuntu 24.04 in vm scaleset is slower then 22.04

### DISCUSSION THREAD

**Discussion by mr3s** [actions/virtual-environments#11115] [General questions]
When I deploy Ubuntu 24.04 to a classic VM, the performance is fine, but when I deploy to a scale set VM, it is up to 150% slower than 22.04 in the same configuration.

24.04 images builded from ubuntu-24.04.pkr.hcl
22.04 images builded from ubuntu-22.04.pkr.hcl

**Comment by MakotoGew** [comment]
Thx

**Comment by mr3s** [comment]
Do you have a similar experience, should I create an issue?
